fireworks/scripts/lpad_run.py

```diff
@@ -702,7 +702,7 @@ def lpad():
     addwf_parser.set_defaults(func=add_wf)
 
     append_wf_parser = subparsers.add_parser('append_wflow', help='append a workflow from file to a workflow on launchpad')
-    append_wf_parser.add_argument('-i', '--fw_id', type=int, nargs='+', help='parent firework ids')
+    append_wf_parser.add_argument(*fw_id_args, type=fw_id_kwargs["type"], nargs=fw_id_kwargs["nargs"], help='parent firework ids')
```

Comment:
I think this code will be easier to read if you just hard-code the ``type`` and ``nargs`` rather than refer to the dict keys

Reply:
(nevermind, this is fine)